in_toto/in_toto_verify.py

```diff
@@ -217,6 +219,7 @@ def create_parser():
     )
 
     parser.add_argument(*GPG_HOME_ARGS, **GPG_HOME_KWARGS)
+    parser.add_argument(*RUN_TIMEOUT_ARGS, **RUN_TIMEOUT_KWARGS)
```

Comment:
minor nit: the help message says
> integer that represents the max timeout in seconds for the for the **in-toto-run command**

This isn't accurate in this context. An easy fix would be to say something like

> step/inspection subprocess timeout in seconds.

A more user friendly approach would be to not re-use the common arg, but create a new one with a dedicated help message. I'm fine either way. Can you update?

Reply:
Yes I didn't think about the help when I checked. Would you recommend I change the argument so it's not `--run-timeout` as well to just `--timeout` or `--verify-timeout`? It would be more in depth change but my thought was the familiarity with the run command.